src/sentry/models/release.py

```diff
@@ -94,6 +94,70 @@ def to_string(cls, value):
             raise ValueError(repr(value))
 
 
+class ReleaseModelManager(models.Manager):
+    @staticmethod
+    def _convert_build_code_to_build_number(build_code):
+        """
+        Helper function that takes the build_code and checks if that build code can be parsed into
+        a 64 bit integer
+        Inputs:
+            * build_code: str
+        Returns:
+            * build_number
+        """
+        build_number = None
+        if build_code is not None:
+            try:
+                if int(build_code).bit_length() <= 63:
+                    build_number = int(build_code)
```

Comment:
I don’t know right now if this could have a leading - but it obviously should not be negative

Reply:
Fixed